packaging/linux/docker_build.sh

```diff
@@ -21,26 +21,25 @@ fi
 
 mode="$1"
 
-here="$(dirname "$BASH_SOURCE")"
+here="$(dirname "${BASH_SOURCE[0]}")"
 
 clientdir="$(git -C "$here" rev-parse --show-toplevel)"
-kbfsdir="$clientdir/../kbfs"
 
 # Run `git fetch` in all the repos we'll share with the container. This
 # prevents an unattended build machine from falling behind over time.
-for repo in "$clientdir" "$kbfsdir" ; do
-  echo "Fetching $repo"
-  git -C "$repo" fetch
-done
+echo "Fetching $clientdir"
+git -C "$clientdir" fetch
 
 # Arrange to share the S3 credentials. We have to do this with a directory
 # instead of sharing the file directly, because the latter only works on Linux.
 s3cmd_temp="$(mktemp -d)"
-cp ~/.s3cfg "$s3cmd_temp"
+cp /keybase/team/keybase.builds.linux/.kbfs_autogit/build-linux/dot_s3cfg "$s3cmd_temp/.s3cfg"
```

Comment:
in retrospect probably a bad idea to depend even more on keybase but we can revisit after we get hotfix out